frontend/src/routes/user/login@user.svelte

```diff
@@ -139,20 +145,20 @@
 				</p>
 				<label class="block pb-2">
 					<span class="text-gray-700">email</span>
-					<input bind:value={email} class="default-input" id="email" />
+					<input bind:value={email} class="mt-1" id="email" />
 				</label>
 				<label class="block ">
 					<span class="text-gray-700">password</span>
 					<input
 						type="password"
 						on:keyup={handleKeyUp}
 						bind:value={password}
-						class="default-input"
+						class="mt-1"
 						id="password"
 					/>
 				</label>
-				<div class="flex flex-row-reverse  pt-4">
-					<button id="login2" class="default-button" type="button" on:click={login}> Login </button>
+				<div class="flex justify-end pt-4">
+					<Button id="login2" on:click={login}>Login</Button>
```

Comment:
Do we have an ID prop on the button ?

Reply:
We jave `{...$$restProps}`, it's newly added.